apps/ledger-live-desktop/tests/userdata/speculos.json

```diff
@@ -97,10 +97,6 @@
           "enabled": true,
           "overridesRemote": true
         },
-        "staxWelcomeScreen": {
-          "enabled": true,
-          "overridesRemote": true
-        },
```

Comment:
This file is not anymore on the repo, the new file is `speculos-tests-app.json` 
A rebase should resolve this issue 😃

Reply:
alright thanks 👍

Reply:
@valber-ledger can you check again please?